i18n/checkLocaleKeys.js

```diff
@@ -21,6 +21,25 @@ const checkLocaleKeys = () => {
 
     const referenceKeys = extractAllKeys(enFileContent)
 
+    //check if in CI/CD
+    if (process.env.NODE_ENV === 'production') {
+        const missingKeysSummary = findAndReportMissingKeysInNonEnFiles(
+            localeFilesWithoutEnFileName, enFileContent, referenceKeys
+        )
+
+        if (missingKeysSummary.length) {
+            console.error('\x1b[31mMissing keys found:\x1b[0m')
```

Comment:
let's add some messaging to the developer to know what to do when there's an error. 
Likely, they should run the script locally then push the updates, so let's say something along those lines.